tasks.py

```diff
@@ -48,7 +50,7 @@ def is_truthy(arg):
 namespace.configure(
     {
         "nautobot_secrets_providers": {
-            "nautobot_ver": "latest",
+            "nautobot_ver": "2.3.1",
```

Comment:
intentional change?

Reply:
Yep, this is what the Cookiecutter has. Most of the other apps had 2.0.0.  This is going to be maintained by Cookiecutter/drift-manager now.